### PR TITLE
feat: Add filteredQuery to tile host data

### PR DESCRIPTION
**Pull request by bryans99**

### CODE REVIEW

**Review by ryandunlavy — APPROVED** [review]
If this works for you, it works for me :)

**Review by jkaster — APPROVED** [review]

**Comment by jkaster** [comment]
I don't think we have support for `/retest` in this repo

**Comment by jkaster** [comment]
I've restarted the failed jobs via GH workflow

File: packages/sdk-node/test/methods.spec.ts [hunk 1]
```diff
@@ -259,7 +259,7 @@ describe('LookerNodeSDK', () => {
   })
 
   describe('downloads', () => {
-    it(
+    it.skip(
```
Comment (jkaster) [inline packages/sdk-node/test/methods.spec.ts:262]: weird this test was/is causing failures

**Review by jkaster — APPROVED** [review]

**Pull request merged**